libs/go/aealite_agent_example/go.mod

```diff
@@ -0,0 +1,10 @@
+module aealite_agent_example
+
+go 1.14
+
+replace aealite => ../aealite
+
+require (
+	aealite v0.0.0-00010101000000-000000000000
+	github.com/sirupsen/logrus v1.8.1
```

Comment:
?

Reply:
requires aealite it self and some logging module to write logs as done in example in aealite README